src/snapred/backend/dao/ingredients/ReductionIngredients.py

```diff
@@ -1,28 +1,19 @@
+from typing import List, Optional
+
 from pydantic import BaseModel
 
-from snapred.backend.dao.ReductionState import ReductionState
-from snapred.backend.dao.RunConfig import RunConfig
+from snapred.backend.dao.GroupPeakList import GroupPeakList
 from snapred.backend.dao.state.PixelGroup import PixelGroup
 
 
 class ReductionIngredients(BaseModel):
-    """Class to hold the instrument configuration."""
-
-    # NOTE these depend on lite mode and focus group through PixelGroup
-    # it is therefore necessary for the reduction service to have
-    # access to the focus group to properly create these
-
-    runConfig: RunConfig
-    reductionState: ReductionState
-    pixelGroup: PixelGroup
+    """Data class to hold the ingredients for each subrecipe of reduction and itself"""
 
-    # if we need specific getter and setter methods, we can use the @property decorator
-    # https://docs.python.org/3/library/functions.html#property
-    #
-    # @property
-    # def key(self) -> str:
-    #     return self._key
+    maskList: List[str]
```

Comment:
should this be a list of `WorkspaceName`s?

Reply:
probably